packages/fsweb/webpack.config.js

```diff
@@ -233,12 +231,10 @@ module.exports = function(env, options) {
     ]);
   } else {
     (!options || !options.json) && console.log('Webpacking for Development');
-    globalConfig.serve = {
-      content: "./dev-server",
-      add: (app, middleware, options) => {
-        app.use(convert(history()));
-      }
```

Comment:
This made it so that you could visit paths besides the root one and have the app still show up. To do the same with webpack-dev-server, you'll need to add `historyApiFallback: true`

Also, please update the yarn.lock file.

Reply:
Option historyApiFallback added to webpack-dev-server configuration. yarn.lock file updated.